src/test/deployments.test.js

```diff
@@ -155,20 +157,24 @@ describe("Deployment component", function () {
   });
 
   it("has the correct deployment id", function () {
-    var cellContent = this.component.props.children[0].props.children;
+    var cellContent = ShallowUtils.getText(this.component.props.children[0]);
     expect(cellContent).to.equal("123");
   });
 
   it("has correct apps in list element", function () {
-    _.each(this.component.props.children[1].props.children.props.children,
-        function (li, i) {
-      expect(li.props.children).to.equal("app" + (i + 1));
+    var list_text = ShallowUtils.getText(this.component.props.children[1].props.children);
+    expect(list_text).to.equal("app1app2app3");
+    // Alternatively, using React's Children API
```

Comment:
See http://facebook.github.io/react/docs/top-level-api.html#react.children and http://facebook.github.io/react/tips/children-props-type.html